src/sentences/dpt.rs

```diff
@@ -0,0 +1,135 @@
+use nom::character::complete::char;
+use nom::combinator::opt;
+use nom::number::complete::double;
+use nom::IResult;
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
+
+use crate::Error;
+use crate::ParseResult;
+use crate::SentenceType;
+
+// DPT - Depth of Water
+//         1   2   3   4
+//         |   |   |   |
+//  $--DPT,x.x,x.x,x.x*hh<CR><LF>
+// Field Number:
+
+// 1. Water depth relative to transducer, meters
+
```

Comment:
You can either remove this empty lines or put a doc comment.
In this case the document will not do much because there is a list, however, when we are talking about text this separates the text with a new line. It will not trigger a new line if they don't have an empty doc comment in between.